sites/skeleton.dev/src/routes/(inner)/utilities/modals/+page.svelte

```diff
@@ -598,6 +598,77 @@ const modalComponent: ModalComponent = {
 				</AccordionItem>
 			</Accordion>
 		</section>
+		<section class="space-y-4">
+			<h2 class="h2">Built-in Responses</h2>
+			<!-- prettier-ignore -->
+			<p>There are 3 built-in response types for modals:</p>
+			<Accordion autocollapse class="card variant-glass p-4">
+				<AccordionItem open>
+					<svelte:fragment slot="summary"><h3 class="h3" data-toc-ignore>Alert Modal</h3></svelte:fragment>
+					<svelte:fragment slot="content">
+						<p>For alert modals, no response will be returned. The modal will simply close when the user clicks the "Cancel" button.</p>
+					</svelte:fragment>
+				</AccordionItem>
+				<AccordionItem>
+					<svelte:fragment slot="summary"><h3 class="h3" data-toc-ignore>Confirm Modal</h3></svelte:fragment>
+					<svelte:fragment slot="content">
+						<p>
+							Confirm modals, as the name implies, will return either <code class="code">true</code> or <code class="code">false</code>.
+							Clicking the "OK" button will return <code class="code">true</code>, while other interactions such as pressing "Cancel" or
+							using the backdrop will return <code class="code">false</code>.
```

Comment:
using the backdrop will return undefined!

Reply:
Great catch, thanks. I wasn't cautious with what I was writing.